source/api/image/2.1/index.md

```diff
@@ -637,7 +637,7 @@ More complex examples are given in the [Complete Response Example](#complete-res
 
 There _MAY_ be one or more services associated with an image. See the [Service Profiles][service-profiles] annex for more information.
 
-The following shows a simple use of `service` to associate geographic metadata with the image.
+The following shows a use of `service` to associate the login page of an authentication system needed to access the image.  For further information, please see [Authentication](#authentication).
 
```

Comment:
It's the image as described in the current info.json response. All of the other 5.x sections use "the image" too. Will try to make it clearer though.